lib/wallet/api/http/Cardano/Wallet/Api/Http/Server/Error.hs

```diff
@@ -481,6 +519,21 @@ instance IsServerError ErrBalanceTx where
                 , fmt $ blockListF' "-" conflictF conflicts
                 ]
 
+instance IsServerError ErrBalanceTxInternalError where
+    toServerError = \case
+        ErrUnderestimatedFee co _st ->
```

Comment:
```suggestion
        ErrUnderestimatedFee c _tx ->
```
- I believe `c` is a pretty standard short variable name for a `Coin`. Arguably it could be something else like `underesimation`, but I don't see how `co` is better.

Reply:
Sure. I don't think I even touched this name, I don't see how `co` is better than `c` either.